cfme/tests/configure/test_tag.py

```diff
@@ -158,8 +160,10 @@ def test_edit_tag(self, appliance, tags):
             test_flag: rest
```

Comment:
what do you think about renaming this test to `test_edit_tag_from_detail` so it will have a unique name (easier to distinguish for collection of the specific test)